packages/mobile/src/components/challenge-rewards-drawer/ChallengeRewardsDrawerContent.tsx

```diff
@@ -102,98 +135,159 @@ export const ChallengeRewardsDrawerContent = ({
+                  strength='strong'
+                  textTransform='uppercase'
+                  variant='label'
+                  size='l'
+                >
+                  {messages.progress}
+                </Text>
+                <ProgressBar progress={currentStep} max={stepCount} />
+              </View>
+            ) : null}
+          </View>
+          <View
             style={[
-              styles.subheader,
-              hasCompleted ? styles.statusTextComplete : null,
-              isInProgress ? styles.statusTextInProgress : null
+              styles.statusCell,
+              hasCompleted ? styles.statusCellComplete : null
             ]}
-            weight='heavy'
-            textTransform='uppercase'
           >
-            {statusText}
-          </Text>
+            <Text
+              style={[
+                styles.subheader,
+                hasCompleted ? styles.statusTextComplete : null,
+                isInProgress ? styles.statusTextInProgress : null
+              ]}
+              strength='strong'
+              textTransform='uppercase'
+              variant='label'
+            >
+              {statusText}
+            </Text>
+          </View>
         </View>
-      </View>
-      {children}
-      <View style={styles.claimRewardsContainer}>
-        {claimableAmount > 0 && onClaim
-          ? [
-              <Text
-                key='claimableAmount'
-                style={styles.claimableAmount}
-                weight='heavy'
-                textTransform='uppercase'
-              >
-                {claimableAmountText}
-              </Text>,
-              <Button
-                key='claimButton'
-                containerStyle={styles.claimButtonContainer}
-                style={styles.claimButton}
-                type={claimInProgress ? ButtonType.COMMON : ButtonType.PRIMARY}
-                disabled={claimInProgress}
-                title={messages.claim}
-                onPress={onClaim}
-                renderIcon={(color) =>
-                  claimInProgress ? (
-                    <LoadingSpinner />
-                  ) : (
-                    <IconCheck fill={color} />
-                  )
-                }
-                iconPosition='left'
-              />
-            ]
-          : null}
-        {claimedAmount > 0 && challengeState !== 'disbursed' ? (
-          <Text style={styles.claimedAmount} weight='heavy'>
-            {claimedAmountText}
-          </Text>
-        ) : null}
-        {claimError ? <ClaimError aaoErrorCode={aaoErrorCode} /> : null}
-      </View>
-    </View>
+        {children}
+        <View style={styles.claimRewardsContainer}>
+          {claimableAmount > 0 && onClaim
+            ? isCooldownChallenge && isRewardsCooldownEnabled
+              ? [renderCooldownSummaryTable()]
+              : [
+                  <Text
+                    key='claimableAmount'
+                    style={styles.claimableAmount}
```

Comment:
similar to above, not sure how to maintain the layout change without making broader changes and opting to leave alone for now because these components should be removed when fully migrated to cooldowns. all new components i made sure to avoid custom styling tho